tasks/app2-1.js

```diff
@@ -1,9 +1,26 @@
 import express from 'express';
 import * as userRouter from './routes/users.js';
+import { logger, errorLogger } from './utils/logger.js';
+import cors from 'cors';
 
 const app = express();
-
 app.listen(3000);
 app.use(express.json());
 
+const corsOptions = { origin: false };
```

Comment:
This looks like a great candidate to be stored in a constants or config file of your application.